CHANGELOG.rst

```diff
@@ -15,6 +15,11 @@ Change Log
 
 Unreleased
 
+[3.8.41]
+--------
+
+* Authenticate tableau user without enterprise
```

Comment:
Is this a bugfix or a feature change?  This description could use more detail.

Reply:
Thank you @nedbat for pointing this. This is a feature change. I will add more detail to it.